database/schema.sql

```diff
@@ -58,7 +62,8 @@ CREATE TABLE "public"."messenger" (
 
 CREATE TABLE "public"."favorite" (
 	"postId" int NOT NULL,
-	"userId" int NOT NULL
+	"userId" int NOT NULL,
+	"isFavorite" BOOLEAN NOT NULL
```

Comment:
This should not have a `isFavorite` because if the row does not exist, then it is not a favorite.